src/templates/_base.njk

```diff
@@ -15,12 +15,12 @@
 
     <body>
         <header>
-            <nav class="navbar navbar-light navbar-expand bg-light mb-3 pt-3 pb-3">
+            <nav class="navbar navbar-light navbar-expand bg-light mb-4 pt-2 pb-2">
                 <div class="container flex-column flex-md-row">
                     <a class="navbar-brand" href="/">
-                        <img src="/svg/logo.svg" alt="Generic Hipster Coffee" width="178" height="178"/>
+                        <img src="/svg/logo.svg" alt="Generic Hipster Coffee" width="180" height="180"/>
                     </a>
-                    <ul class="navbar-nav gap-sm-4">
+                    <ul class="navbar-nav gap-lg-3">
```

Comment:
![image](https://user-images.githubusercontent.com/334494/191000555-41b0aaa4-bb41-4dba-b09f-2b4e5037aa09.png)
asi by to sneslo dříve větší mezery

Reply:
![image](https://user-images.githubusercontent.com/334494/191000756-31c749aa-3131-4a91-a5f5-a69755065ad7.png)
vlastne i tady